### PR TITLE
[FR] Cleanup `Play_style`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for cleanup the French translation of the wiki page `Play_style`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Play_style/fr.md [hunk 1]
```diff
@@ -22,17 +18,17 @@ Ce style de jeu est largement considéré comme la méthode la plus difficile à
 
 La double souris est un style de jeu dans lequel un joueur a une souris dans chaque main ; il utilise une souris pour taper et une autre pour déplacer le curseur dans le jeu.
 
-Ce style de jeu était considéré comme une blague par la plupart des joueurs jusqu'en mai 2013, lorsque le joueur canadien ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) a commencé à obtenir les meilleurs scores sur des maps très respectées comme [Hatsune Miku - With a Dance Number (val0108)](https://osu.ppy.sh/beatmapsets/29691) et [The Quick Brown Fox - The Big Black (Blue Dragon)](https://osu.ppy.sh/beatmapsets/41823) en jouant avec deux souris.
+Ce style de jeu était considéré comme une blague par la plupart des joueurs jusqu'en mai 2013, lorsque le joueur canadien ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) a commencé à obtenir les meilleurs scores sur des beatmaps très respectées comme [Hatsune Miku - With a Dance Number (val0108)](https://osu.ppy.sh/beatbeatmapsets/29691) et [The Quick Brown Fox - The Big Black (Blue Dragon)](https://osu.ppy.sh/beatbeatmapsets/41823) en jouant avec deux souris.
```
Comment (Walavouchey) [inline wiki/Play_style/fr.md:21]: i see you've done a search and replace, but that broke these links

```suggestion
Ce style de jeu était considéré comme une blague par la plupart des joueurs jusqu'en mai 2013, lorsque le joueur canadien ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) a commencé à obtenir les meilleurs scores sur des beatmaps très respectées comme [Hatsune Miku - With a Dance Number (val0108)](https://osu.ppy.sh/beatmapsets/29691) et [The Quick Brown Fox - The Big Black (Blue Dragon)](https://osu.ppy.sh/beatmapsets/41823) en jouant avec deux souris.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Play_style/fr.md (hunk 1 above)
Comment (DoryanR) [reply wiki/Play_style/fr.md:21]: Oops, I hadn't seen that, thanks!

**Review by Biokax — CHANGES_REQUESTED** [review]

**Comment by Biokax** [comment]
wait for @Ganondorf-1 review before merging

File: wiki/Play_style/fr.md [hunk 2]
```diff
@@ -42,13 +38,13 @@ Certains joueurs n'utilisent que le clavier pour taper, tandis que d'autres ne p
 
 *Remarque : Ce style de jeu nécessite l'utilisation d'une [tablette graphique](/wiki/Gameplay/Input_device/Graphics_tablet).*
 
-Les joueurs sur tablette font passer un stylet au-dessus d'une tablette graphique pour naviguer dans le curseur du jeu et ne tapent les coups qu'en tapant le stylet sur la surface de la tablette graphique elle-même.
+Les joueurs sur tablette font passer un stylet au-dessus d'une tablette graphique pour naviguer avec le curseur et ne tapent les coups qu'en tapant le stylet sur la surface de la tablette graphique elle-même.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:41]:
```suggestion
Les joueurs sur tablette font passer un stylet au-dessus d'une tablette graphique pour naviguer avec le curseur et ne cliquent sur les objets qu'en tapant avec le stylet sur la surface de la tablette graphique elle-même.
```

File: wiki/Play_style/fr.md [hunk 3]
```diff
@@ -42,13 +38,13 @@ Certains joueurs n'utilisent que le clavier pour taper, tandis que d'autres ne p
...
 
 *Remarque : La superposition d'entrée montrera toutes les touches de la tablette comme `M1`.*
 
 #### Tapx
 
-La communauté considère qu'il est pratiquement impossible de réaliser des stream en continu avec une seule entrée tactile à un BPM élevé. Ainsi, pour faire des stream en continu en tant que joueur sur tablette uniquement, un joueur doit alterner entre le tapotement de son stylo sur la tablette et l'appui sur une touche secondaire assignée (généralement une touche de la tablette ou du clavier). Cette méthode est parfois appelée *tapx*.
+La communauté considère qu'il est pratiquement impossible de réaliser des streams en continu avec une seule entrée tactile à un BPM élevé. Ainsi, pour faire des streams en continu en tant que joueur sur tablette uniquement, un joueur doit alterner entre le tapotement de son stylo sur la tablette et l'appui sur une touche secondaire assignée (généralement une touche de la tablette ou du clavier). Cette méthode est parfois appelée *tapx*.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:47]:
```suggestion
La communauté considère qu'il est pratiquement impossible de réaliser des streams en continu avec une seule entrée tactile à un BPM élevé. Ainsi, pour faire des streams en continu en tant que joueur sur tablette uniquement, un joueur doit alterner entre le tapotement de son stylet sur la tablette et l'appui sur une touche secondaire assignée (généralement une touche de la tablette ou du clavier). Cette méthode est parfois appelée *tapx*.
```

File: wiki/Play_style/fr.md [hunk 4]
```diff
@@ -42,13 +38,13 @@ Certains joueurs n'utilisent que le clavier pour taper, tandis que d'autres ne p
...
 
 *Remarque : Comme le stylet a été considéré comme `M1`, seul le bouton de tapotement secondaire fonctionnera, et `X` est généralement le bouton de tapotement secondaire par défaut.*
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:49]:
```suggestion
*Remarque : Comme le stylet est considéré comme `M1`, seul le bouton de clic secondaire fonctionnera, et `X` est généralement le bouton de clic secondaire par défaut.*
```

File: wiki/Play_style/fr.md [hunk 5]
```diff
@@ -58,32 +54,32 @@ La communauté considère qu'il est pratiquement impossible de réaliser des str
 
 La tablette et le clavier sont un style de jeu populaire parmi de nombreux joueurs passionnés, compétitifs et de haut niveau.
 
-Les joueurs munis d'une tablette ou d'un clavier "survolent" ou "font glisser" le stylet de la tablette pour naviguer dans le curseur du jeu et tapent à l'aide du clavier ou des boutons de la tablette.
+Les joueurs munis d'une tablette ou d'un clavier "survolent" ou "font glisser" le stylet de la tablette pour naviguer avec le curseur et tapent à l'aide du clavier ou des boutons de la tablette.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:57]:
```suggestion
Les joueurs munis d'une tablette ou d'un clavier font "hover" (survoler) ou "drag" (font glisser) le stylet de la tablette pour naviguer avec le curseur et tapent à l'aide du clavier ou des boutons de la tablette.
```

File: wiki/Play_style/fr.md [hunk 6]
```diff
@@ -58,32 +54,32 @@ La communauté considère qu'il est pratiquement impossible de réaliser des str
...
+- **Joystick** : Une façon créative de contrôler le curseur d'osu!. Dans ce style de jeu, le joueur tient et déplace un joystick pour faire bouger le curseur et, selon la conception du joystick du joueur, tape en utilisant les boutons de déclenchement/de face du joystick, ou avec les touches du clavier pour enregistrer les battements. Ce style de jeu est largement considéré comme une sorte de "défi" en raison des inconvénients évidents et de l'habileté requise pour contrôler le mouvement du joystick.
 - **Plaque métallique** : Il s'agit d'une méthode spéciale pour remplacer le clavier pour taper des rythmes. La méthode a été introduite par ![][flag_US] [geckogates](https://osu.ppy.sh/users/geckogates) et nécessite un peu de câblage et de connaissances électroniques pour la construire. Le joueur attachera un fil aux doigts qui cliquent (connectés au circuit du clavier pour `Z` et `X`) et touchera les plaques de métal pour enregistrer les battements.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:81]:
```suggestion
- **Plaque métallique** : Il s'agit d'une méthode spéciale qui remplace le clavier pour taper. La méthode a été introduite par ![][flag_US] [geckogates](https://osu.ppy.sh/users/geckogates) et nécessite un peu de câblage et de connaissances électroniques pour la construire. Le joueur attachera un fil aux doigts qui cliquent (connectés au circuit du clavier pour `Z` et `X`) et touchera les plaques de métal pour enregistrer les battements.
```

File: wiki/Play_style/fr.md [hunk 7]
```diff
@@ -58,32 +54,32 @@ La communauté considère qu'il est pratiquement impossible de réaliser des str
...
-- **Manette de jeu** : Tout comme le joystick, cette méthode est considérée comme une façon créative et originale de jouer à osu!. Selon la marque de la manette, le joueur peut utiliser un trackpad intégré, un stick de contrôle/analogique ou un D-Pad pour contrôler le curseur du jeu. Certains joueurs trouveront plus confortable d'utiliser les boutons déjà présents sur la manette pour frapper, ou d'utiliser un clavier ordinaire.
+- **Manette de jeu** : Tout comme le joystick, cette méthode est considérée comme une façon créative et originale de jouer à osu!. Selon la marque de la manette, le joueur peut utiliser un trackpad intégré, un stick de contrôle/analogique ou un D-Pad pour contrôler le curseur. Certains joueurs trouveront plus confortable d'utiliser les boutons déjà présents sur la manette pour frapper, ou d'utiliser un clavier ordinaire.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:82]:
```suggestion
- **Manette de jeu** : Tout comme le joystick, cette méthode est considérée comme une façon créative et originale de jouer à osu!. Selon la marque de la manette, le joueur peut utiliser un trackpad intégré, un stick de contrôle/analogique ou un D-Pad pour contrôler le curseur. Certains joueurs trouveront plus confortable d'utiliser les boutons déjà présents sur la manette pour cliquer, ou d'utiliser un clavier ordinaire.
```

File: wiki/Play_style/fr.md [hunk 8]
```diff
@@ -97,53 +93,53 @@ Il existe deux styles principaux de réglage de l'emplacement des touches sur le
 
 *Note : Pour les raccourci clavier, la valeur par défaut est `Z` `X` `C` `V` (KDDK, ou BRRB).*
 
-### Manette Taiko
+### Manette taiko
 
 Le manette taiko est le style de jeu et la manette par défaut de la série [Taiko no Tatsujin](https://fr.wikipedia.org/wiki/Taiko_no_Tatsujin) ([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) pour les États-Unis).
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:98]:
```suggestion
La manette taiko est le style de jeu et la manette par défaut de la série [Taiko no Tatsujin](https://fr.wikipedia.org/wiki/Taiko_no_Tatsujin) ([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) pour les États-Unis).
```

File: wiki/Play_style/fr.md [hunk 9]
```diff
@@ -97,53 +93,53 @@ Il existe deux styles principaux de réglage de l'emplacement des touches sur le
...
 Le clavier est le style de jeu le plus couramment utilisé et par défaut pour osu!catch.
 
-Pour jouer avec un clavier, les joueurs appuient sur les touches fléchées gauche et droite pour déplacer l'attrapeur et maintiennent la touche `leftShift` enfoncée pour se déplacer.
+Pour jouer avec un clavier, les joueurs appuient sur les flèches gauche et droite pour déplacer le catcher et maintiennent la touche `leftShift` enfoncée pour se déplacer.
 
-Il est possible de modifier les touches utilisées ; certains joueurs apprécient les touches `A`/`D` pour les déplacements et la barre d'espace ou la touche `rightShift` pour les dash.
+Il est possible de modifier les touches utilisées ; certains joueurs apprécient les touches `A`/`D` pour les déplacements et la barre d'espace ou la touche `rightShift` pour les dashs.
 
-La superposition des touches montre que les touches fléchées sont saisies comme `L` et `R` et le bouton du dash comme `D`.
+La superposition des touches montre que les flèches sont saisies comme `L` et `R` et le bouton du dash comme `D`.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:120]:
```suggestion
Le compteur de clics montre que les flèches sont saisies comme `L` et `R` et le bouton du dash comme `D`.
```

File: wiki/Play_style/fr.md [hunk 10]
```diff
@@ -97,53 +93,53 @@ Il existe deux styles principaux de réglage de l'emplacement des touches sur le
...
 
 ### Clavier et souris
 
-Clavier et souris est un style de jeu alternatif mineur du clavier, dans lequel les joueurs peuvent effectuer des dashs en utilisant des clics de souris à la place. Bien que certains joueurs préfèrent l'inverse et préfèrent contrôler les mouvements de gauche et de droite en utilisant les boutons gauche/droit de la souris et activer les dashs avec le clavier.
+Le clavier et la souris est un style de jeu alternatif mineur du clavier, dans lequel les joueurs peuvent effectuer des dashs en utilisant des clics de souris à la place. Bien que certains joueurs préfèrent l'inverse et préfèrent contrôler les mouvements de gauche et de droite en utilisant les boutons gauche/droit de la souris et activer les dashs avec le clavier.
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:124]:
```suggestion
Le clavier et la souris sont un style de jeu alternatif mineur du clavier, dans lequel les joueurs peuvent effectuer des dashs en utilisant des clics de souris à la place. Bien que certains joueurs préfèrent l'inverse et préfèrent contrôler les mouvements de gauche et de droite en utilisant les boutons gauche/droit de la souris et activer les dashs avec le clavier.
```

File: wiki/Play_style/fr.md [hunk 11]
```diff
@@ -97,53 +93,53 @@ Il existe deux styles principaux de réglage de l'emplacement des touches sur le
...
 
-Pour dash, de nombreux joueurs utilisent l'un des boutons de leur manette de jeu. Cependant, certains joueurs préfèrent utiliser des gâchettes, des bumpers ou des sticks.
+Pour le dash, de nombreux joueurs utilisent l'un des boutons de leur manette de jeu. Cependant, certains joueurs préfèrent utiliser des gâchettes, des bumpers ou des sticks.
 
 ### Autres styles
 
-- **Souris uniquement** : Le jeu à la souris est un style de jeu de niche dans lequel les joueurs contrôlent tous les mouvements et les dash de l'attrapeur uniquement par le biais de la souris. Cette option n'est toutefois disponible que sur les souris d'ordinateur dotées des boutons M4 et M5 (qui se trouvent généralement sur le côté gauche de la souris).
+- **Souris uniquement** : Le jeu à la souris est un style de jeu dans lequel les joueurs contrôlent tous les mouvements et les dash du catcher uniquement par le biais de la souris. Cette option n'est toutefois disponible que sur les souris d'ordinateur dotées des boutons M4 et M5 (qui se trouvent généralement sur le côté gauche de la souris).
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:142]:
```suggestion
- **Souris uniquement** : Jouer uniquement à la souris est un style de jeu dans lequel les joueurs contrôlent tous les mouvements et les dashs du catcher uniquement par le biais de la souris. Cette option n'est toutefois disponible que sur les souris d'ordinateur dotées des boutons M4 et M5 (qui se trouvent généralement sur le côté gauche de la souris).
```

File: wiki/Play_style/fr.md [hunk 12]
```diff
@@ -22,17 +18,17 @@ Ce style de jeu est largement considéré comme la méthode la plus difficile à
...
 ### Souris et clavier
 
 La souris et le clavier sont le style de jeu le plus populaire dans osu! car ils utilisent les périphériques informatiques les plus basiques et sollicitent moins la main qui tient la souris.
 
 Les joueurs avec souris et clavier jouent presque exactement de la même manière que les joueurs avec souris uniquement, à la seule différence que deux touches du clavier (`Z` et `X` par défaut) sont utilisées pour taper au lieu de la souris elle-même.
 
-Certains joueurs n'utilisent que le clavier pour taper, tandis que d'autres ne passent au clavier qu'en cas de streaming (action de taper une série très rapide d'objets en succession rapide) et utilisent la souris pour cliquer pour tout le reste.
+Certains joueurs n'utilisent que le clavier pour taper, tandis que d'autres ne passent au clavier qu'en cas de stream (action de taper une série très rapide d'objets en succession rapide) et utilisent la souris pour cliquer pour tout le reste.
 
 *Note : La superposition des touches montre les pressions du clavier comme `K1` et `K2`.*
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:33]:
```suggestion
*Note : Le compteur de clics montre les pressions du clavier comme `K1` et `K2`.*
```

File: wiki/Play_style/fr.md [hunk 13]
```diff
@@ -42,13 +38,13 @@ Certains joueurs n'utilisent que le clavier pour taper, tandis que d'autres ne p
 
 *Remarque : Ce style de jeu nécessite l'utilisation d'une [tablette graphique](/wiki/Gameplay/Input_device/Graphics_tablet).*
 
-Les joueurs sur tablette font passer un stylet au-dessus d'une tablette graphique pour naviguer dans le curseur du jeu et ne tapent les coups qu'en tapant le stylet sur la surface de la tablette graphique elle-même.
+Les joueurs sur tablette font passer un stylet au-dessus d'une tablette graphique pour naviguer avec le curseur et ne tapent les coups qu'en tapant le stylet sur la surface de la tablette graphique elle-même.
 
 *Remarque : La superposition d'entrée montrera toutes les touches de la tablette comme `M1`.*
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:43]:
```suggestion
*Remarque : Le compteur de clics montrera toutes les touches de la tablette comme `M1`.*
```

File: wiki/Play_style/fr.md [hunk 14]
```diff
@@ -58,32 +54,32 @@ La communauté considère qu'il est pratiquement impossible de réaliser des str
...
+Pour déplacer le curseur, les joueurs utilisant un écran tactile touchent l'écran de leur appareil avec un stylet ou leur doigt à l'endroit précis où ils veulent que le curseur se trouve. Le curseur "saute" alors à l'endroit touché.
 
-Lorsqu'un joueur à écran tactile touche l'écran, son appareil déplace simultanément le curseur vers ce point et envoie une entrée de souris. Ainsi, pour toucher, suivre les sliders et faire tourner les spinners, les joueurs à écran tactile ne toucheront les objets que lorsqu'ils voudront qu'ils le soient.
+Lorsqu'un joueur avec un écran tactile touche l'écran, son appareil déplace simultanément le curseur vers ce point et envoie une entrée de souris. Ainsi, pour toucher, suivre les sliders et faire tourner les spinners, les joueurs avec un écran tactile ne toucheront les objets que lorsqu'ils voudront qu'ils le soient.
 
 Ce style de jeu peut ressembler à celui de [osu!stream](/wiki/osu!stream) en raison de l'action de toucher directement l'écran plutôt que de déplacer un curseur.
 
 *Remarque : La superposition des touches indique l'entrée tactile comme `M1`.*
```
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:71]:
```suggestion
*Remarque : Le compteur de clics indique l'entrée tactile comme `M1`.*
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Biokax — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**